system-int-test/test/cmr/system_int_test/search/granule_shapefile_search_test.clj

```diff
@@ -19,15 +21,22 @@
 
 (use-fixtures :each (ingest/reset-fixture {"provguid1" "PROV1"}))
 
-(defn polygon
+(defn- polygon
   "Creates a single ring polygon with the given ordinates. Points must be in counter clockwise order.
   The polygon will be closed automatically."
   [& ords]
   (poly/polygon [(apply umm-s/ords->ring ords)]))
 
+(def formats
+  "Shapfile formats to be tested"
+  {
```

Comment:
nit: Clojure style is: `{"ESRI"...` with no blank line before "ESRI".

Reply:
Fixed